sync/awsdatabricks.py

```diff
@@ -218,7 +220,7 @@ def get_cluster_report(
 
 def _get_cluster_report(
     cluster_id: str, plan_type: str, compute_type: str, allow_incomplete: bool
-) -> Response[DatabricksClusterReport]:
+) -> Response[DatabricksClusterReport | dict]:
```

Comment:
why did I add this?.. 🤔

Reply:
Ah, I think this is because Pycharm is confused about what is actually being returned in the event of an error -

<img width="1161" alt="Screenshot 2023-06-01 at 9 26 36 PM" src="https://github.com/synccomputingcode/syncsparkpy/assets/117235708/a9bfdcb2-9b49-429a-b1f8-43763f7fafc8">